src/Config.php

```diff
@@ -76,9 +76,28 @@ public function __construct($version, $apiKey, $apiVersion)
     {
         $this->setVersion($version);
 
-        $this->setApiKey($apiKey ?: getenv('STRIPE_API_KEY'));
+        $this->setApiKey($apiKey ?: self::getEnvVariable('STRIPE_API_KEY', ''));
 
-        $this->setApiVersion($apiVersion ?: getenv('STRIPE_API_VERSION') ?: '2017-06-05');
+        $this->setApiVersion($apiVersion ?: self::getEnvVariable('STRIPE_API_VERSION', '2017-06-05'));
+    }
+
+    /**
+     * @param string      $name
+     * @param string|null $default
+     *
+     * @return string|null
+     */
+    private static function getEnvVariable($name, $default = null)
+    {
+        if (isset($_SERVER[$name])) {
+            return $_SERVER[$name];
+        }
+
+        if (PHP_SAPI === 'cli' && ($value = getenv($name) !== false)) {
+            return $value;
```

Comment:
This returns a boolean, since the condition above is performing a check after the assignment.

It seems the closing `)` needs to be earlier

```php
if (PHP_SAPI === 'cli' && ($value = getenv($name)) !== false) {
```

want to do this change or do it a tad differently?

Reply:
Yes, I just noticed this too! Fixed. :)